scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import yaml, os, shutil
+
+dir = os.path.dirname(__file__)
+
+if os.path.exists('{}/validation_messages'.format(dir)):
+    shutil.rmtree('{}/validation_messages'.format(dir))
```

Comment:
This feels a bit dangerous. Should we warn that the folder exists and exit instead? Or maybe ask for the path to save files to.